### DISCUSSION TITLE
The pipe flow case with different cross-sectional areas

### DISCUSSION THREAD

**Discussion by ahmedelhanafy2022** [UvaCsl/HemoCell#48] [General discussion]
Hello, Hemocell community
This is my first participation in this helpful discussion.
I have an inquiry about the validity of Hemocell calculations ( max vel. and rel app. viscosity) in the **pipe flow case** with different cross-sectional areas, not just a straight pipe. Does anyone have an idea about this?
Thanks in advance
Ahmed

**Comment by gzavo** [comment]
From my part I don't know of any work that has investigated such question in detail.
